bindgen/src/lib.rs

```diff
@@ -86,10 +86,21 @@ impl uniffi_bindgen::BindingGenerator for BindingGenerator {
     ) -> anyhow::Result<()> {
         let bindings_file = out_dir.join(format!("{}.cs", ci.namespace()));
         let mut f = File::create(&bindings_file)?;
+
+        let version = env!("CARGO_PKG_VERSION");
+        let header = format!(
+            r#"// <auto-generated>
```

Comment:
It feels like this should be part of [Wrapper.cs](https://github.com/NordSecurity/uniffi-bindgen-cs/blob/main/bindgen/templates/wrapper.cs). `CARGO_PKG_VERSION` can be added as `version` to `CsWrapper` struct, and accessed from the template using `{{ version }}`.